shell/platform/tizen/flutter_tizen_elementary.cc

```diff
@@ -52,3 +50,48 @@ FlutterDesktopViewRef FlutterDesktopViewCreateFromNewWindow(
 
   return HandleForView(view.release());
 }
+
+FlutterDesktopViewRef FlutterDesktopViewCreateFromElmParent(
+    const FlutterDesktopViewProperties& view_properties,
+    FlutterDesktopEngineRef engine,
+    void* parent) {
+  std::unique_ptr<flutter::TizenViewBase> tizen_view =
+      std::make_unique<flutter::TizenViewElementary>(
+          view_properties.width, view_properties.height,
+          static_cast<Evas_Object*>(parent));
+
+  auto view =
+      std::make_unique<flutter::FlutterTizenView>(std::move(tizen_view));
+
+  // Take ownership of the engine, starting it if necessary.
+  view->SetEngine(
+      std::unique_ptr<flutter::FlutterTizenEngine>(EngineFromHandle(engine)));
+  view->CreateRenderSurface();
+  if (!view->engine()->IsRunning()) {
+    if (!view->engine()->RunEngine()) {
+      return nullptr;
+    }
+  }
+
+  view->SendInitialGeometry();
+
+  return HandleForView(view.release());
+}
+
+void* FlutterDesktopViewGetEvasObject(FlutterDesktopViewRef view_ref) {
+  auto* view = reinterpret_cast<flutter::FlutterTizenView*>(view_ref);
+  if (view->tizen_view()->GetType() == flutter::TizenViewType::kView) {
+    auto* tizen_view =
+        reinterpret_cast<flutter::TizenView*>(view->tizen_view());
+    return tizen_view->GetRenderTargetContainer();
+  }
+  return nullptr;
+}
+
+void FlutterDesktopViewResize(FlutterDesktopViewRef view_ref,
+                              int32_t width,
+                              int32_t height) {
+  auto* view = reinterpret_cast<flutter::FlutterTizenView*>(view_ref);
+  flutter::TizenGeometry view_geometry = {0, 0, width, height};
+  view->tizen_view()->OnGeometryChanged(view_geometry);
```

Comment:
In my opinion, the embedder APIs related to View should be something like C wrapper of FlutterTizenView interface.
so, I believe that to provide the resizing, it has to work internally through FlutterTizenView's interface.

This is the basic design I was initially thinking of, so it's open for discussion and you don't need to fix it right away. I'm curious about your opinion.

Reply:
I don't agree. `FlutterTizenView` is simply an abstraction layer for a Flutter view and it should not be considered as a wrapper of `TizenView`.

Reply:
> `FlutterTizenView` is simply an abstraction layer for a Flutter view and it should not be considered as a wrapper of `TizenView`.

What you said is a simple principle I came up with when I first designed it. 
When I first designed FlutterDeskopViewXXX APIs, I had the following design in mind.
```
void FlutterDesktopViewXXX()
{
   ...
    view->XXX()
   ...
}
```

Currently, FlutterTizenView may have different detailed behaviors depending on TizenViewType. 
So I think this idea may not be suitable. That's why I was curious about your thoughts :)

Reply:
I agree to reduce the abstraction level (depth) of required `FlutterDesktopViewXXX`.
In the case of `resize` rather than simple property change, I think it is better to handle it with `view->Rsize()`.
If the ViewType check is handled inside the FlutterTizenView, I think it is easy to fix when the type check is no longer needed through refactoring.